typescript/phoenix-sdk/tests/watch.ts

```diff
@@ -3,25 +3,30 @@ import { Connection } from "@solana/web3.js";
 import * as Phoenix from "../src";
 
 export async function watch() {
-  const connection = new Connection("https://qn-devnet.solana.fm/");
-  const phoenix = await Phoenix.Client.create(connection, "devnet");
+  const connection = new Connection("http://127.0.0.1:8899");
+  const phoenix = await Phoenix.Client.create(connection, "localhost");
 
-  const market = phoenix.markets.find((market) => market.name === "wSOL/USDC");
+  console.log(phoenix.markets);
+  const market = Array.from(phoenix.markets.values()).find(
+    (market) => market.name === "SOL/USDC"
+  );
   if (!market) throw new Error("Market not found");
 
+  const marketAddress = market.address.toBase58();
+
   let lastLadder: Phoenix.UiLadder | null = null;
   let updates = 0;
   while (updates < 10) {
-    const ladder = market.getUiLadder();
+    const ladder = phoenix.getUiLadder(marketAddress);
     if (JSON.stringify(ladder) !== JSON.stringify(lastLadder)) {
       console.clear();
       console.log("Ladder update", updates + 1, "of", 10, "\n");
-      market.printLadder();
+      phoenix.printLadder(marketAddress);
       lastLadder = ladder;
       updates++;
     }
 
-    await market.refresh(connection);
+    await phoenix.refreshMarket(marketAddress);
```

Comment:
not sure how i feel about this interface. you can have a clock out of sync with a different market